### ISSUE TITLE
Wrapper properties vm cant read  the  exposed properties  

### ISSUE THREAD

**Issue by NidusP** [vuejs/vue-test-utils-next#1111]
Comp `setup` return a render function,  and `expose` specific properties
```jsx
export default defineComponent({
    setup(prop, { expose }){
        const str = ref('hello'), domRef = ref(null);
        const handleClick = () => {
          console.log('handleClick')
        }
        expose({
          handleClick,
          str,
          domRef
        })
        return () => (<div ref={ domRef } onClick={ handleClick } id="div">{ str.value }</div>)
      }
})
```
I cant get `str` `handleClick` ... it is version problem? 
```js
import Comp from './comp';
test('comp', () => {
  it('vm', () => {
    const wrapper = mount(Comp)
    console.log(wrapper.vm.handleClick)
  })
})
```

**Comment by cexbrayat** [comment]
Hi @NidusP 

You'll need to give us more details to help you: what version are you using? What is failing?
If you're using the latest VTU version (rc.17 at the moment), it should work. 

If you have an issue with the latest version, you can either:
- try to provide us a minimal reproduction in a github repo so we can take a look
- add a failing unit test to this repository (see `expose.spec.ts` for all the expose related tests)

**Comment by NidusP** [comment]
Thanks for your help!  I write the test according to the guide and it passed !  (rc.17 and rc.9)

The failed tests environment  were built by someone else,  I looked at the dependencies,  such as `jest` `@vue/test-utils` `babel-jest` `@types/jest`.  So  I tried to add `@vue/vue3-jest` to the dependencies and updated `jest.config.js` , but  still failed. 

Does it concern `@vue/vue3-jest`? I am not very clear about `jest.config.js` details  and the relationships  between dependencies.  The test maybe pass if  `jest.config.js`  is configured correctly ?  I'll look into it again.
